commit 0b398d69acde3377dfbbeb8a4cfe87ae8c8562fa
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Sat Feb 1 23:16:11 2020 +0000

    binutils: drop redundant 'program_name' definition (-fno-common)
    
            * coffdump.c (program_name): Drop redundant definition.
            * srconv.c (program_name): Likewise
            * sysdump.c (program_name): Likewise
    
    Signed-off-by: Sergei Trofimovich <siarheit@google.com>

diff --git a/binutils/ChangeLog b/binutils/ChangeLog
index a7fd43176b..2caa7e950f 100644
--- binutils/ChangeLog
+++ binutils/ChangeLog
@@ -1,3 +1,9 @@
+2020-02-03  Sergei Trofimovich  <siarheit@google.com>
+
+	* coffdump.c (program_name): Drop redundant definition.
+	* srconv.c (program_name): Likewise
+	* sysdump.c (program_name): Likewise
+
 2020-02-02  H.J. Lu  <hongjiu.lu@intel.com>
 
 	PR gas/25380
diff --git a/binutils/coffdump.c b/binutils/coffdump.c
index 531a4e46c3..336da57ca2 100644
--- binutils/coffdump.c
+++ binutils/coffdump.c
@@ -456,8 +456,6 @@ coff_dump (struct coff_ofile *ptr)
     dump_coff_section (ptr->sections + i);
 }
 
-char * program_name;
-
 static void
 show_usage (FILE *file, int status)
 {
diff --git a/binutils/srconv.c b/binutils/srconv.c
index 5742b16759..f071794f0a 100644
--- binutils/srconv.c
+++ binutils/srconv.c
@@ -1687,8 +1687,6 @@ prescan (struct coff_ofile *otree)
     }
 }
 
-char *program_name;
-
 ATTRIBUTE_NORETURN static void
 show_usage (FILE *ffile, int status)
 {
diff --git a/binutils/sysdump.c b/binutils/sysdump.c
index d433e71ed9..7eebbd61d3 100644
--- binutils/sysdump.c
+++ binutils/sysdump.c
@@ -633,8 +633,6 @@ module (void)
     }
 }
 
-char *program_name;
-
 ATTRIBUTE_NORETURN static void
 show_usage (FILE *ffile, int status)
 {
